authors/apps/authentication/urls.py

```diff
@@ -1,11 +1,16 @@
+from django.conf.urls import url
 from django.urls import path
 
 from .views import (
-    LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
+    LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView, ResetPasswordEmail, ResetPasswordConfirm
 )
 
 urlpatterns = [
-    path('user/', UserRetrieveUpdateAPIView.as_view(), name ="updateRetrieve"),
+    path('user/', UserRetrieveUpdateAPIView.as_view(), name="updateRetrieve"),
     path('users/', RegistrationAPIView.as_view(), name='register'),
     path('users/login/', LoginAPIView.as_view(), name='login'),
+
+    path('password_reset/', ResetPasswordEmail.as_view(), name='password_reset'),
+    url(r'^reset/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
```

Comment:
@ManuelDominic, could you also put this API route to the Django 2 format as such `path('reset/<str:uidb64>/<str:token>/')`